driver/mounter_test.go

```diff
@@ -0,0 +1,110 @@
+package driver
+
+import (
+	"errors"
+	"fmt"
+	"testing"
+
+	"github.com/sirupsen/logrus"
+)
+
+type testAttachmentValidator struct {
+	readFileFunc     func(name string) ([]byte, error)
+	evalSymlinksFunc func(path string) (string, error)
+}
+
+func (av *testAttachmentValidator) readFile(name string) ([]byte, error) {
+	return av.readFileFunc(name)
+}
+
+func (av *testAttachmentValidator) evalSymlinks(path string) (string, error) {
+	return av.evalSymlinksFunc(path)
+}
+
+func Test_mounter_IsAttached(t *testing.T) {
+	t.Run("could not evaluate the symbolic link", func(t *testing.T) {
+		m := &mounter{
+			log:      logrus.NewEntry(logrus.New()),
+			kMounter: nil,
+			attachmentValidator: &testAttachmentValidator{
+				readFileFunc: nil,
```

Comment:
If we stub this function out to return a different error, then it becomes clear how the test failed our implementations happens to accidentally call the wrong one. Right now, if we were to hit `readFileFunc`, the code would panic, which is harder to debug.

(I know that the code currently is simple enough for this to be very unlikely, but it's a good practice to account for the (test) maintainability of possible extensions, e.g., refactorings/additions to the logic.)